src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/DeleteTasksControllerTest.java

```diff
@@ -0,0 +1,164 @@
+import uk.gov.hmcts.reform.wataskmanagementapi.SpringBootIntegrationBaseTest;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.restrict.ClientAccessControlService;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.CFTTaskState;
+import uk.gov.hmcts.reform.wataskmanagementapi.cft.query.TaskResourceCaseQueryBuilder;
+import uk.gov.hmcts.reform.wataskmanagementapi.clients.CamundaServiceApi;
+import uk.gov.hmcts.reform.wataskmanagementapi.clients.IdamWebApi;
+import uk.gov.hmcts.reform.wataskmanagementapi.clients.RoleAssignmentServiceApi;
+import uk.gov.hmcts.reform.wataskmanagementapi.controllers.request.DeleteCaseTasksAction;
+import uk.gov.hmcts.reform.wataskmanagementapi.controllers.request.DeleteTasksRequest;
+import uk.gov.hmcts.reform.wataskmanagementapi.domain.camunda.SecurityClassification;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.TaskResource;
+import uk.gov.hmcts.reform.wataskmanagementapi.entity.TaskRoleResource;
+import uk.gov.hmcts.reform.wataskmanagementapi.services.CFTTaskDatabaseService;
+import uk.gov.hmcts.reform.wataskmanagementapi.utils.ServiceMocks;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.post;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.CFTTaskState.TERMINATED;
+import static uk.gov.hmcts.reform.wataskmanagementapi.cft.enums.CFTTaskState.UNASSIGNED;
+import static uk.gov.hmcts.reform.wataskmanagementapi.config.SecurityConfiguration.SERVICE_AUTHORIZATION;
+import static uk.gov.hmcts.reform.wataskmanagementapi.utils.ServiceMocks.IDAM_AUTHORIZATION_TOKEN;
+import static uk.gov.hmcts.reform.wataskmanagementapi.utils.ServiceMocks.SERVICE_AUTHORIZATION_TOKEN;
+
+public class DeleteTasksControllerTest extends SpringBootIntegrationBaseTest {
+    @MockBean
+    private IdamWebApi idamWebApi;
+    @MockBean
+    private CamundaServiceApi camundaServiceApi;
+    @MockBean
+    private RoleAssignmentServiceApi roleAssignmentServiceApi;
+    @MockBean
+    private AuthTokenGenerator authTokenGenerator;
+    @MockBean
+    private ClientAccessControlService clientAccessControlService;
+    @MockBean
+    private ServiceAuthorisationApi serviceAuthorisationApi;
+    @Autowired
+    private CFTTaskDatabaseService cftTaskDatabaseService;
+
+    private ServiceMocks mockServices;
+
+    @BeforeEach
+    void setUp() {
+        when(authTokenGenerator.generate())
+                .thenReturn(IDAM_AUTHORIZATION_TOKEN);
+        mockServices = new ServiceMocks(
+                idamWebApi,
+                serviceAuthorisationApi,
+                camundaServiceApi,
+                roleAssignmentServiceApi
+        );
+    }
+
```

Comment:
As discussed, we should use any of the candidate tests here for the CapturedOutput where we are verifying our logging of untermintated tasks being deleted.

Reply:
Done

Reply:
Done